### ISSUE TITLE
Set learning rate of `GradientBoosting`

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Stdlib#168]
### Is your feature request related to a problem?

It's not possible to set the learning rate of gradient boosting.

### Desired solution

* Add a new parameter `learning_rate: float` to the initializer of `safeds.ml.classification.GradientBoosting` and `safeds.ml.regression.GradientBoosting`
* Raise a value error if `learning_rate` <= 0
* Pass it along to `learning_rate` of the wrapped `scikit-learn` model in the `fit` method
